smalltalksrc/Slang/TMethod.class.st

```diff
@@ -3275,6 +3293,20 @@ TMethod >> usesVariableUninlinably: argName in: aCodeGen [
 						  subNode isVariable and: [ subNode name = argName ] ] ] ] ] ]
 ]
 
+{ #category : #accessing }
+TMethod >> validateVariableDeclarationExists: aVariableName [
+
+	| shouldCheckVariable |
+	shouldCheckVariable := (aVariableName beginsWithAnyOf:
+		                       #( #self , #cascade , #toDoLimit )) not.
```

Comment:
Little syntax error here, it works by chance :)
Literal arrays are either:

```
#( #a #b #c )
{ #a . #b . #c}
```

Here with `#( #self , #cascade , #toDoLimit )` you are creating an array with 5 elements:
 - `#self`
 - `#,`
 - `#cascade`
 - `#,`
 - `#toDoLimit`

And of course, this works because we never have a variable named `#,`